scripts/tests/test_pi_vcc_validation_cli.py

```
@@ -0,0 +1,83 @@
+import tempfile
+import unittest
+from pathlib import Path
+
+ROOT = Path(__file__).resolve().parents[2]
+REAL_HOST = ROOT / "scripts/pi-vcc-real-host-integration.ts"
+SOAK = ROOT / "scripts/run-pi-vcc-continuation-soak.sh"
+
+
+class PiVccValidationCliTest(unittest.TestCase):
+    def run_command(self, command, env=None):
+        return subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)